Documentation/unit-files-and-scheduling.md

```diff
@@ -57,6 +57,35 @@ To use instance units, simply create a unit file whose name matches the `<name>@
 
 When working with instance units, it is strongly recommended that all units be _entirely homogenous_. This means that any unit created as, say, `foo@1.service`, should be created only from the unit named `foo@.service`. This homogeneity will be enforced by the fleet API in future.
 
+## Definition of the Install Section
+
+Unit files which have `[Install]` section will be automatically enabled by fleet. This means that states of such unit files may not be tracked by fleet. For example you have loaded `my.service` unit file:
```

Comment:
s/which have/which have an/
s/states/the states/
s/may not/cannot/
"For example, assume we have loaded this `my.service` unit file:"